skipruntime-ts/core/src/api.ts

```diff
@@ -318,6 +318,8 @@ export interface LazyCompute<K extends Json, V extends Json> {
  * Skip Runtime internal state.
  */
 export interface Context extends Managed {
+  readonly session: (Json & DepSafe) | null;
```

Comment:
Exposing the session on the Context seems great, big improvement on exposing the collection directly!

This could use a docstring explaining what it is/how to use it, though.

Reply:
One important aspect is just when it is null vs non-null.